<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>579</width>
    <height>369</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::ActionsContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QHBoxLayout" name="layout_menu">
     <item>
      <widget class="QGroupBox" name="gbMenu">
       <property name="title">
        <string>Главное меню:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="btOpen">
          <property name="text">
           <string>Открыть файл</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btSave">
          <property name="text">
           <string>Сохранить в файл</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btSort">
          <property name="text">
           <string>Сортировка</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btDeleteItem">
          <property name="text">
           <string>Удалить запись</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QGroupBox" name="gbAdd">
       <property name="title">
        <string>Добавление и редактирование записи:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="laCompany">
            <property name="text">
             <string>Производитель:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="edCompany"/>
          </item>
          <item>
           <widget class="QLabel" name="laDiagonal">
            <property name="text">
             <string>Диагональ:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="edDiagonal"/>
          </item>
          <item>
           <widget class="QPushButton" name="btCreateItem">
            <property name="text">
             <string>Добавить запись</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QToolButton" name="toolButton">
            <property name="text">
             <string>X</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="laColor">
            <property name="text">
             <string>Цвет:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="edColor"/>
          </item>
          <item>
           <widget class="QLabel" name="laCost">
            <property name="text">
             <string>Цена:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="edCost"/>
          </item>
          <item>
           <widget class="QPushButton" name="btEdit">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Редактировать запись</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="tableTV">
     <property name="inputMethodHints">
      <set>Qt::ImhNone</set>
     </property>
     <property name="sortingEnabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
